docs/search/get-started-with-search/search-page/navigate-messages-in-search-results.md

```diff
@@ -6,54 +6,53 @@ description: When you run a search query, messages display in the Message, Aggre
 
 
 
-When you run a search your results are provided in a **Messages** tab. If the search conducted an aggregation you'll also get an **Aggregates** tab.
+When you run a search, the results are displayed in the **Messages** tab. If the search includes an aggregation, an **Aggregates** tab will also be displayed.
 
 ![Messages and Aggregates tab.png](/img/search/get-started-search/search-page/messages-aggregates-tab.png)
 
-The **Signatures** tab is shown when using [LogReduce](/docs/search/logreduce).
+The **Signatures** tab is shown when using [LogReduce](/docs/search/logreduce).
 
 ![Signatures tab.png](/img/search/get-started-search/search-page/signatures-tab.png)
 
-## Column adjustments
+## Column Adjustments
```

Comment:
Reverting edit. Our style is to use Sentence Case (see [Style Guide](https://help.sumologic.com/docs/contributing/style-guide/#capitalization)).

```suggestion
## Column adjustments
```